skimage/util/arraypad.py

```diff
@@ -1469,3 +1469,43 @@ def pad(array, pad_width, mode=None, **kwargs):
             newmat = _pad_wrap(newmat, (pad_before, pad_after), axis)
 
     return newmat
+
+
+def crop(ar, crop_width, copy=False, order='K'):
+    """Crop array `ar` by `crop_width` along each dimension.
+
+    Parameters
+    ----------
+    ar : array-like of rank N
+        Input array.
+    crop_width : {sequence, int}
+        Number of values to remove from the edges of each axis.
+        ((before_1, after_1), ... (before_N, after_N)) specifies unique
+        crop widths at the start and end of each axis.
+        ((before, after),) specifies the same start and end crops for
+        every axis.
+        (int,) or int is a shortcut for before = after = int for all
+        axes.
+    copy : bool, optional
+        Ensure that the returned array is contiguous. Normally, a crop
```

Comment:
But a crop may also return a contiguous copy, so really this has nothing to do with contiguity but whether we have a view on the original data or not.